app/Http/Controllers/Internlink/InternlinkController.php

```diff
@@ -74,6 +90,39 @@ public function save(Request $request)
         }
     }
 
+    public function viewInternship($id)
+    {
+        $internship = Internship::where('id', $id)->get()->first();
+        $link = $internship->link;
+        $member = $link->member;
+
+        $member_details = ['name' => $member->full_name];
+
+        if($link->show_uk_phone == 1){
+            $member_details = array_merge($member_details, ['mobile_uk' => $member->mobile_uk]);
+        }
+        if($link->show_home_phone == 1){
+            $member_details = array_merge($member_details, ['mobile_home' => $member->mobile_home]);
+        }
+        if($link->show_hermes_email){
```

Comment:
If you use boolean for the field columns, `$link->show_home_phone` will probably work. Also, that line could be replaced with `$member_details['mobile_uk'] = $member->mobile_uk`.